pkg/controller/cronjob/cronjob_controllerv2.go

```diff
@@ -48,6 +48,7 @@ import (
 	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/controller/cronjob/metrics"
 	"k8s.io/utils/pointer"
```

Comment:
Same here. Let's drop pointer on the files that you are updating.

Reply:
thanks reply.  it is done.